src/class-parsely.php

```diff
@@ -1788,7 +1788,10 @@ private function get_author_name( ?WP_User $author ): string {
 	private function get_author_names( WP_Post $post ): array {
 		$authors = $this->get_coauthor_names( $post->ID );
 		if ( empty( $authors ) ) {
-			$authors = array( get_user_by( 'id', $post->post_author ) );
+			$post_author = get_user_by( 'id', $post->post_author );
+			if ( $post_author ) {
+				$authors = array( get_user_by( 'id', $post->post_author ) );
```

Comment:
```suggestion
				$authors = array( $post_author );
```

Does this change make sense and avoid repeating the `get_user_by()` call?